com.greyrook.siptop.json

```diff
@@ -0,0 +1,78 @@
+{
+  "app-id": "com.greyrook.siptop",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "19.08",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "siptop.sh",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--device=all",
+    "--filesystem=xdg-documents:ro",
+    "--filesystem=xdg-pictures:ro",
+    "--filesystem=xdg-music:ro",
+    "--filesystem=xdg-videos:ro",
+    "--filesystem=xdg-download",
+    "--filesystem=xdg-run/dconf",
+    "--filesystem=~/.config/dconf:ro",
+    "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+    "--talk-name=ca.desrt.dconf",
+    "--talk-name=org.freedesktop.Notifications",
+    "--env=XDG_CURRENT_DESKTOP=Unity",
+    "--talk-name=org.kde.StatusNotifierWatcher"
+  ],
+  "modules": [
+    {
+      "name": "siptop",
+      "buildsystem": "simple",
+      "build-commands": [
+        "cp -r siptop-0.1.0/* ${FLATPAK_DEST}/bin/",
+        "install siptop.sh ${FLATPAK_DEST}/bin/siptop.sh",
+        "install -Dm644 com.greyrook.siptop.appdata.xml ${FLATPAK_DEST}/share/appdata/${FLATPAK_ID}.appdata.xml",
+        "install -Dm644 com.greyrook.siptop.icon.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png",
+        "install -Dm644 com.greyrook.siptop.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop",
+        "desktop-file-edit --set-key=\"Icon\" --set-value=\"com.greyrook.siptop\" ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop",
+        "desktop-file-edit --set-key=\"Exec\" --set-value=\"siptop.sh %U\" ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop",
+        "desktop-file-edit --set-key=\"StartupWMClass\" --set-value=\"siptop\" ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop",
+        "desktop-file-edit --set-key=\"X-Flatpak-RenamedFrom\" --set-value=\"siptop.desktop;\" ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop",
```

Comment:
Why not have these in the desktop file directly?

Reply:
Is it a problem to have it there? The manifest is based on the one from Slack. I'm not really sure why this is there instead of in the desktop file.

Reply:
Slack is an extra-data app, meaning that files are changed after Slack binary is downloaded as we don't have redistribution rights.